tests/system/test_client.py

```diff
@@ -2286,12 +2293,17 @@ def test_nested_table_to_arrow(self):
         self.assertTrue(pyarrow.types.is_list(record_col[1].type))
         self.assertTrue(pyarrow.types.is_int64(record_col[1].type.value_type))
 
-    def temp_dataset(self, dataset_id, location=None):
+    def temp_dataset(self, dataset_id, *args, **kwargs):
         project = Config.CLIENT.project
         dataset_ref = bigquery.DatasetReference(project, dataset_id)
         dataset = Dataset(dataset_ref)
-        if location:
-            dataset.location = location
+        if kwargs.get("location"):
+            dataset.location = kwargs.get("location")
+        if kwargs.get("max_time_travel_hours"):
+            dataset.max_time_travel_hours = kwargs.get("max_time_travel_hours")
```

Comment:
I see you have also opened #1683 that covers `max_time_travel_hours`. Let's make these changes over there, and confine this PR to only `default_rounding_mode` stuff.

Reply:
@Linchin  done. cherry pick the commit to that pull request.

Reply:
Could you also delete these lines in this PR? Thank you!